dotcom-rendering/src/components/TextBlockComponent.stories.tsx

```diff
@@ -223,3 +227,37 @@ export const longWordStory = () => {
 	);
 };
 longWordStory.storyName = 'Long Words';
+
+export const startingWithLinkStory = () => {
+	return (
+		<div css={containerStyles}>
+			<TextBlockComponent
+				html={startingWithLink}
+				format={{
+					theme: Pillar.News,
+					design: ArticleDesign.Standard,
+					display: ArticleDisplay.Standard,
+				}}
+				isFirstParagraph={false}
```

Comment:
This is not really testing anything, you’d need an `ArticleDisplay.Immersive` and `isFirstParagraph` for the drop cap to happen at all!

Reply:
oh lol, thanks! fixing..